game/src/OptionMenu/VideoTab.gd

```diff
@@ -2,8 +2,27 @@ extends HBoxContainer
 
 @export var initial_focus: Control
 
+@onready var revert_dialog = get_node("KeepChangesDialog")
+@onready var resolution_selector = get_node("VBoxContainer/GridContainer/ResolutionSelector")
```

Comment:
Please do not rely on `get_node`, always prefer to use export nodes when you need reference things, as its less prone to breaking on refactoring and it does not require messing with code to fix it via Godot's interface.